src/_playground/documentation/Properties/_PropertyTable.js

```diff
@@ -9,15 +10,15 @@ import React from 'react';
 import sortBy from 'sort-by';
 import Table from '../../../Table/Table';
 
-const PropertyTable = ({ title, propTypes, defaultProps, propDescriptions }) => {
+const PropertyTable = ({ title, propTypes = {}, defaultProps = {}, propDescriptions = {} }) => {
     if (!propTypes) {
         return (
             <em>This component has no defined properties.</em>
         );
     }
 
     let propInfo = Object.keys(propTypes).map(propName => {
-        return { propName: propName, required: propTypes[propName].typeRequired };
+        return { propName: propName, required: !!propTypes[propName].typeRequired };
```

Comment:
Force a boolean to prevent sorting issues (if `typeRequired` was undefined rather than false, props were sorting at the bottom for what appeared to be no reason).